<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phase_coherence_window</class>
 <widget class="QMainWindow" name="phase_coherence_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1783</width>
    <height>1055</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="main_layout" stretch="9,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="20">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="4,7" columnstretch="7,2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <widget class="QGroupBox" name="verticalGroupBox_2">
              <property name="title">
               <string>Group 1</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="SignalPlot" name="plot_1a" native="true"/>
               </item>
               <item>
                <widget class="SignalPlot" name="plot_1b" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="verticalGroupBox">
              <property name="title">
               <string>Group 2</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="SignalPlot" name="plot_2a" native="true"/>
               </item>
               <item>
                <widget class="SignalPlot" name="plot_2b" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Wavelet Transform Options</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QGroupBox" name="freqOptions">
                 <property name="title">
                  <string>Frequency</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_2">
                  <property name="labelAlignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="formAlignment">
                   <set>Qt::AlignHCenter|Qt::AlignTop</set>
                  </property>
                  <item row="1" column="1">
                   <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLabel" name="label_3">
                      <property name="frameShape">
                       <enum>QFrame::NoFrame</enum>
                      </property>
                      <property name="text">
                       <string>Max Fr</string>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLineEdit" name="line_fmax">
                      <property name="minimumSize">
                       <size>
                        <width>70</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>70</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="1">
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLabel" name="label_4">
                      <property name="frameShape">
                       <enum>QFrame::NoFrame</enum>
                      </property>
                      <property name="text">
                       <string>Min Fr</string>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLineEdit" name="line_fmin">
                      <property name="maximumSize">
                       <size>
                        <width>76</width>
                        <height>22</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="3" column="1">
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLabel" name="label_5">
                      <property name="frameShape">
                       <enum>QFrame::NoFrame</enum>
                      </property>
                      <property name="text">
                       <string>Res</string>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLineEdit" name="line_res">
                      <property name="maximumSize">
                       <size>
                        <width>70</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QGroupBox" name="groupBox">
                   <property name="title">
                    <string>Cut Edges</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_18">
                    <item row="0" column="0">
                     <widget class="QRadioButton" name="radio_cut_on">
                      <property name="text">
                       <string>On</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QRadioButton" name="radio_cut_off">
                      <property name="text">
                       <string>Off</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox">
                   <property name="title">
                    <string>Preprocessing</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_13">
                    <item row="0" column="0">
                     <widget class="QRadioButton" name="radio_preproc_on">
                      <property name="text">
                       <string>On</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QRadioButton" name="radio_preproc_off">
                      <property name="text">
                       <string>Off</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_3">
                   <property name="title">
                    <string>Wavelet Type</string>
                   </property>
                   <layout class="QFormLayout" name="formLayout_4">
                    <item row="0" column="1">
                     <widget class="QComboBox" name="combo_wavelet_type">
                      <item>
                       <property name="text">
                        <string>Lognorm</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Morlet</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Bump</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Group Coherence Options</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QGroupBox" name="verticalGroupBox">
                 <property name="title">
                  <string>Percentile</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <item>
                   <widget class="QLineEdit" name="line_percentile">
                    <property name="text">
                     <string>95</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0,0">
               <item>
                <widget class="QGroupBox" name="groupbox_stats">
                 <property name="title">
                  <string>Statistics</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_16">
                  <item>
                   <widget class="QListWidget" name="list_stat"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>fmin:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="line_stat_fmin"/>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>fmax:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="line_stat_fmax"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QPushButton" name="btn_stat_add">
                      <property name="text">
                       <string>Add freq interval</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_stat_del">
                      <property name="text">
                       <string>Delete freq interval</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btn_stat_calc">
                    <property name="text">
                     <string>Calculate statistics</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string>Calculate group coherence</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <widget class="QPushButton" name="btn_calculate_all">
                      <property name="text">
                       <string>Transform All</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_calculate_single">
                      <property name="text">
                       <string>Transform Single</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QProgressBar" name="progress">
                    <property name="value">
                     <number>24</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lbl_progress">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="verticalGroupBox">
                 <property name="title">
                  <string>Log</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_18">
                  <item>
                   <widget class="QPlainTextEdit" name="text_log">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,2">
             <item>
              <widget class="QGroupBox" name="horizontalGroupBox_3">
               <property name="title">
                <string>Coherence</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="AmplitudePlot" name="plot_right" native="true"/>
                </item>
                <item>
                 <widget class="GroupCoherencePlot" name="plot_main" native="true"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="horizontalGroupBox_4">
               <property name="title">
                <string>Statistical test</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QTableView" name="tbl_stat"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_6" rowstretch="1,0">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="advanced_options_section"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1783</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignalPlot</class>
   <extends>QWidget</extends>
   <header>gui.plotting.plots.SignalPlot</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AmplitudePlot</class>
   <extends>QWidget</extends>
   <header>gui.plotting.plots.AmplitudePlot</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GroupCoherencePlot</class>
   <extends>QWidget</extends>
   <header>gui.plotting.plots.GroupCoherencePlot</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
